pkg/ctl/get/iamserviceaccount.go

```diff
@@ -74,44 +69,9 @@ func doGetIAMServiceAccount(cmd *cmdutils.Cmd, serviceAccount *api.ClusterIAMSer
 	}
 
 	stackManager := ctl.NewStackManager(cfg)
+	iamServiceAccountManager := iam.New(cfg.Metadata.Name, ctl, stackManager, nil, nil)
+	serviceAccounts, err := iamServiceAccountManager.Get(namespace, name)
 
-	remoteServiceAccounts, err := stackManager.GetIAMServiceAccounts()
-	if err != nil {
-		return errors.Wrap(err, "getting iamserviceaccounts")
-	}
-	// we will show user the object based on given config file,
-	// and what we have learned about the iamserviceaccounts;
-	// that is not ideal, but we don't have a better option yet
-	cfg.IAM.ServiceAccounts = []*api.ClusterIAMServiceAccount{}
-
-	saFilter := filter.NewIAMServiceAccountFilter()
-
-	if cmd.ClusterConfigFile == "" {
-		// reset defaulted fields to avoid output being a complete lie
-		cfg.VPC = nil
-		cfg.CloudWatch = nil
-		// only return the iamserviceaccount that user asked for
-		var notFoundErr error
-		if serviceAccount.Name != "" { // name was given
-			notFoundErr = fmt.Errorf("iamserviceaccount %q not found", serviceAccount.NameString())
-			saFilter.AppendIncludeNames(serviceAccount.NameString())
-		} else if cmd.CobraCommand.Flag("namespace").Changed { // only namespace was given
-			notFoundErr = fmt.Errorf("no iamserviceaccounts found in namespace %q", serviceAccount.Namespace)
-			err = saFilter.AppendIncludeGlobs(remoteServiceAccounts, serviceAccount.Namespace+"/*")
```

Comment:
does this mean globs aren't supported?

Reply:
yes I think this has removed that functionality, i'm happy to bring it back. 

Before I do that I think its worth noting that the `--help` output shows: `--namespace string     namespace to look for iamserviceaccount (default "default")`, so I'm not sure if anyone ever knew it was even supported. I'm sort of open to just not adding it back in 🤷‍♂️

Reply:
i think we can probably live without it 🤷‍♀️